manifest.json

```diff
@@ -1,18 +1,15 @@
 {
-  "manifest_version": 2,
+  "manifest_version": 3,
```

Comment:
Is 3 a valid value?  I'm not sure this makes sense to increase from 2 to 3, it looks like this might be intended to stay set to 2 based on the [manifest file format page](https://developer.chrome.com/apps/manifest).